docs/modules/usage/runtimes.md

```diff
@@ -0,0 +1,77 @@
+# Runtime Configuration
+
+A Runtime is an environment where the OpenHands agent can edit files and run
+commands.
+
+By default, OpenHands uses a Docker-based runtime, running on your local computer.
+But this can be tricky to set up and manage, so we also offer support for "remote" runtimes.
+In these cases, you'll generally need to set up an account with a runtime provider
+and get an API key to use their runtime.
+
+## Docker Runtime
+This is the default Runtime that's used when you start OpenHands. You might notice
+some flags being passed to `docker run` that make this possible:
+
+```
+docker run # ...
+    -e SANDBOX_RUNTIME_CONTAINER_IMAGE=ghcr.io/all-hands-ai/runtime:0.11-nikolaik \
+    -v /var/run/docker.sock:/var/run/docker.sock \
+    # ...
+```
+
+The `SANDBOX_RUNTIME_CONTAINER_IMAGE` from nikolaik is a pre-built runtime image
+that contains our Runtime server, as well as some basic utilities for Python and NodeJS.
+You can also [build your own runtime image](how-to/custom-sandbox-guide).
+
+### Connecting to your filesystem
+One useful feature here is the ability to connect to your local filesystem.
+This functionality can be tricky, so we don't recommend it by default. But
+it allows OpenHands to work directly on files on your computer.
+
+To mount your filesystem into the runtime, add the following options to
+the `docker run` command:
```

Comment:
Same here. "Tricky" doesn't say what is tricky. Maybe a better framing would be: there are advantages and disadvantages - if that's what we want to say. Or we could just remove the idea of _recommending_ here, consider documentation to be the place to inform, not necessarily to recommend.

Reply:
Thanks, tried to get more descriptive. I do think it's important to give the user a sense for what tradeoffs there are here, and where the footguns are